hazelcast-documentation/src/HazelcastSimulator/ExecutingASimulatorTest.md

```diff
@@ -4,17 +4,311 @@
 
 After you install and prepare the Hazelcast Simulator for your environment, it is time to perform a test.
 
-The following steps execute a Hazelcast Simulator test.
+In the following sections, we provide an example test and its output along with the required edits to the files `simulator.properties` and `test.properties`.
 
-1. Install the Hazelcast Simulator.
-2. Create a folder85 for your tests. Let's call it your working folder.
-3. Copy the `simulator.properties` file from the `/conf` folder of the Hazelcast Simulator to your working folder.
-4. Edit the `simulator.properties` file according to your needs.
-5. Copy the `test.properties` file from the `/simulator-tests` folder of Hazelcast Simulator to your working folder.
-6. Edit the `test.properties` file according to your needs.
-5. Execute the `run.sh` script while you are in your working folder to perform your Simulator test.
+### Creating and editing properties file
 
-In the following sections, we provide an example test and its output along with the required edits to the files `simulator.properties` and `test.properties`.
+***NOTE:*** *For a full description of the file `simulator.properties`, please see the [Simulator.Properties File Description section](#simulator-properties-file-description). Also you can find sample simulator properties in the `dist/simulator-tests/simulator.properties`. You can copy the this file to working directory ,then you can edit according to your needs.*
+
+You need to give the classpath of `test` in the file `test.properties` as shown below.
+
+```
+example@class=com.hazelcast.simulator.tests.ExampleTest
```

Comment:
You can find ExampleTest in the simulator repo's test module. Imho ExampleTest more suitable than StringStringMapTest for user use simulator first time. He/She can easily understand what is going on here.